script/release.sh

```diff
@@ -134,6 +135,13 @@ git_pull() {
   cd ..
 }
 
+
+git_sync() {
+  cd $CLI
+  git pull upstream
```

Comment:
I've tried  and got:
```
You asked to pull from the remote 'upstream', but did not specify
a branch. Because this is not the default configured remote
for your current branch, you must specify a branch on the command line.
```